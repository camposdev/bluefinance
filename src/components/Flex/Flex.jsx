import React from 'react';
import PropTypes from "prop-types";
import styled from "styled-components";

const Wrapper = styled.div`
  display: flex;
  justify-content: ${({ justify }) => justify};
  align-items: ${({ align }) => align};
`;

const Flex = props => <Wrapper {...props} />;

Flex.propTypes = {
  justify: PropTypes.string,
  align: PropTypes.string
};

Flex.defaultProps = {
  justify: "space-between",
  align: "center"
};

export default Flex;
