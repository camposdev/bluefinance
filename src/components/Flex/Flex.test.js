import React from "react";
import renderer from "react-test-renderer";
import Flex from "./Flex";

describe("Flex component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Flex><div>1</div><div>2</div></Flex>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
