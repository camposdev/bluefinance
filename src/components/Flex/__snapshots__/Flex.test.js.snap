// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex component renders correctly 1`] = `
<div
  className="sc-bdVaJa UTvpt"
>
  <div>
    1
  </div>
  <div>
    2
  </div>
</div>
`;
